mkdocs.yml

```diff
@@ -217,7 +217,48 @@ nav:
           - '<code>server.py</code>': 'question/server.md'
           - 'Accessibility': 'question/accessibility.md'
       - 'Question runtime environment': 'questionRuntime/index.md'
-      - 'Elements for writing questions': 'elements.md'
+      - 'Elements for writing questions':
+          - elements/index.md
+          - '<span class="muted">pl-</span>answer-panel': 'elements/pl-answer-panel.md'
+          - '<span class="muted">pl-</span>big-o-input': 'elements/pl-big-o-input.md'
+          - '<span class="muted">pl-</span>card': 'elements/pl-card.md'
+          - '<span class="muted">pl-</span>checkbox': 'elements/pl-checkbox.md'
+          - '<span class="muted">pl-</span>code': 'elements/pl-code.md'
+          - '<span class="muted">pl-</span>dataframe': 'elements/pl-dataframe.md'
+          - '<span class="muted">pl-</span>drawing': 'pl-drawing/index.md'
```

Comment:
Since this is added here, should the other link to this page a few lines below be removed?

Reply:
I'm not opposed, I wasn't sure if calling that page out specifically was useful. Probably not?